src/components/views/WelcomeSection.vue

```diff
@@ -10,12 +10,12 @@ export default defineComponent({
 
 <template>
   <TheContainer class="of-welcome-section">
-    <div class="welcome-section">
-      <div class="welcome-message">
+    <div class="welcome">
+      <div class="welcome__message">
         <SectionTitle size="large">
             Welcome to the best cinema in Monterail
         </SectionTitle>
-        <p class="welcome-text">
+        <p class="welcome__message--text">
```

Comment:
Looks better! I had same problem with BEM. My classes were too long and I was confused with `--`. Here is comment straight from my old PR  ⬇

> As you know the `--` sign is used to create modifiers classes `Flags on blocks or elements. Use 
> them to change appearance, behavior or state`. For example if moviePoster content should be red 
> then we can create a modifier class for it like this `movie__details-poster---red` where `---red` 
> changes the appearance of the movie poster.

Additionally there is article that should help you with long classes 🙂
https://scalablecss.com/bem-nesting-grandchild-elements/

Reply:
Ok, so here it should rather be "welcome__message__text", right?

Reply:
I will give you approve. If you feel like doing it on this branch then do so, but I think it can be done on separate branch either. It is up to you 😉

Reply:
Thanks a loooot! This article is a treasure! ✨ Maybe there's a chance that I wouldn't hate BEM so much 😉 Here I will go for "welcome__main-text" then.